source/content/guides/localdev/04-troubleshoot-support.md

```diff
@@ -64,9 +64,9 @@ The steps in this section should only be used as a last resort. This resets Loca
 
 While Localdev uses parts of Lando under the hood, they should be considered separate tools. Localdev will not respect existing Lando configuration files, and we recommend keeping the two tools separate. That is, if you are going to use both tools, they should point to separate project directories.
 
-### Can WordPress Site Networks be developed through Localdev?
+### Can WordPress Multisites be developed through Localdev?
 
-At this time, WordPress Site Network (also known as WordPress Multisite) is supported only for networks using subdirectories, not subdomains. See the [Choose Between Subdirectories and Subdomains](/guides/multisite/considerations/#choose-between-subdirectories-and-subdomains) section of our [WordPress Site Networks](/guides/multisite) guide for more information.
+At this time, WordPress Multisite is supported only for networks using subdirectories, not subdomains. See the [Choose Between Subdirectories and Subdomains](/guides/multisite/considerations/#choose-between-subdirectories-and-subdomains) section of our [WordPress Multisite](/guides/multisite) guide for more information.
```

Comment:
This line is factually incorrect. People can (and do) use subdomain multisite on the platform. In fact, the [linked considerations section](https://pantheon.io/docs/guides/multisite/considerations/#choose-between-subdirectories-and-subdomains) explains the difference between the two on Pantheon but does not actually say "subdomain multisite is not supported". I think this line could be removed or replaced with something to the effect of:

```suggestion
See the [Choose Between Subdirectories and Subdomains](/guides/multisite/considerations/#choose-between-subdirectories-and-subdomains) section of our [WordPress Multisite](/guides/multisite) guide for more information.
```